pkts-sip/src/test/java/io/pkts/packet/sip/impl/SipParserTest.java

```diff
@@ -133,23 +134,50 @@ public void testConsumeUserInfoHostNoUserPart() throws Exception {
      */
     @Test
     public void testConsumeUserInfoHostInvalidHostChars() throws Exception {
-        assertConsumeUserInfoHostThrowsParseException("//127.0.0.1:5060");
-        assertConsumeUserInfoHostThrowsParseException("//app.example.com");
-        assertConsumeUserInfoHostThrowsParseException("app.example.com/");
-        assertConsumeUserInfoHostThrowsParseException("hostname/");
-        assertConsumeUserInfoHostThrowsParseException("");
-        assertConsumeUserInfoHostThrowsParseException("     ");
-        assertConsumeUserInfoHostThrowsParseException(".");
-        assertConsumeUserInfoHostThrowsParseException(".com");
-        assertConsumeUserInfoHostThrowsParseException("9");
-        assertConsumeUserInfoHostThrowsParseException("@:;");
-        assertConsumeUserInfoHostThrowsParseException("@ip.pbx.com");
-        assertConsumeUserInfoHostThrowsParseException("hello there!");
-        assertConsumeUserInfoHostThrowsParseException("customer.ip.pbx.com/");
-        assertConsumeUserInfoHostThrowsParseException("customer.ip.pbx.com/test");
-        assertConsumeUserInfoHostThrowsParseException("127:0:0:1");
-        assertConsumeUserInfoHostThrowsParseException("127.0.0.1:");
-        assertConsumeUserInfoHostThrowsParseException("127.0.0.1000");
+        assertConsumeUserInfoHostThrowsParseException("//customer.ip.pbx.com",
+                "The SIP URI does not specify a valid host. Error encountered after 0 characters.");
```

Comment:
I would suggest not to validate the actual message itself (since it could change, be internationalized etc) but rather that the error offset is the expected one.